### DISCUSSION TITLE
Possible to conditionally install packages depending on host?

### DISCUSSION THREAD

**Discussion by rieje** [wbthomason/packer.nvim#910] [General]
Is it possible to conditionally install packages depending on host (e.g. shell variable $HOSTNAME)? I only use dark theme on my desktop machine and only use light theme on my laptop. I can of course install both and do e.g. `if os.getenv("HOME")` and set the theme, but I would rather not install packages I wouldn't use on that machine. I also use prefer to use the same dotfiles for ease of maintenance (or at least not have any files used by both be different, I'm okay with them having unique files dedicated to themes since they will never conflict). for both machines since this is my current setup and nothing else is different otherwise.

Otherwise, is this a reasonable feature request?

**Comment by ViViDboarder** [comment]
You can do `disable = os.getenv("HOME") ~= "blah"` to disable the plugin if not the right environment.
